main/inc/lib/exercise_show_functions.lib.php

```diff
@@ -364,7 +364,7 @@ public static function display_unique_or_multiple_answer(
                 break;
         }
 
-        $icon = in_array($answerType, [UNIQUE_ANSWER, UNIQUE_ANSWER_NO_OPTION]) ? 'radio' : 'checkbox';
+        $icon = in_array($answerType, [UNIQUE_ANSWER, UNIQUE_ANSWER_NO_OPTION, QUESTION_PT_TYPE_CATEGORY_RANKING]) ? 'radio' : 'checkbox';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 138 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)